netbox/templates/dcim/inc/rack_elevation.html

```diff
@@ -27,13 +27,13 @@
                     {% ifequal u.device.face face_id %}
                         <a href="{% url 'dcim:device' pk=u.device.pk %}" data-toggle="popover" data-trigger="hover" data-container="body" data-html="true"
                            data-content="{{ u.device.device_role }}<br />{{ u.device.device_type.full_name }} ({{ u.device.device_type.u_height }}U){% if u.device.asset_tag %}<br />{{ u.device.asset_tag }}{% endif %}">
-                            {{ u.device.name|default:u.device.device_role }}
+                            {{ u.device|default:u.device.device_role }}
```

Comment:
We want to look specifically at `name` in this case, because `__str__` calls `display_name()`, which will return the device type if no name is set.